### DISCUSSION TITLE
Running brew installed limactl with shared network

### DISCUSSION THREAD

**Discussion by maver1ck** [lima-vm/lima#687] [Q&A]
### Description

Hi,
I have following problem.
I'm trying to use shared network in lima.
Unfortunately I'm getting following error.
```
INFO[0000] Using the existing instance "default"
FATA[0000] networks.yaml field `paths.vdeSwitch` error: file "/opt/homebrew/Cellar/vde/2.3.2_1/bin/vde_switch" is not owned by "root" (uid: 0), but by uid 501
```
What can I do ?

**Comment by spkane** [comment]
Those files must be owned by root according to the vdeswitch documentation. You can either uninstall it via homebrew and built it yourself, or fix the homebrew formula so that it sets the permissions correctly.

https://github.com/lima-vm/vde_vmnet#step-1-install-vde-2-vde_switch
